analysis/filter/StructureModifier/README.md

```diff
@@ -7,6 +7,7 @@ The Structure Modifier is used to modify the structure of .cc.json files. It ena
 - Remove nodes from a project: The resulting project will not include these nodes and their children.
 - Declare a node as root: This means that the chosen node will become the root node of the resulting sub-project.
 - Move nodes within the project: All children of the source node will be transferred to the destination node.
+- Rename the mcc metric to complexity or sonar_complexity (they have previously been renamed to mcc).
```

Comment:
You mean " (they have previously been renamed to MCC)" or "(they have previously been named to MCC)"?